modules/token-overview.adoc

```diff
@@ -0,0 +1,66 @@
+
+* *Administer Repositories*. When selected, provides the user administrator access to all repositories to which the granting user or robot account has access.
+
+* *Create Repositories*.  When selected, provides the user the ability to create repositories in any namespaces that the granting user or robot account is allowed to create repositories. 
+
+* *View all visible repositories*. When selected, provides the user the ability to view and pull all repositories visible to the granting user or robot account.
+
+* *Read/Write to any accessible repositories*.  When selected, provides the user the ability to view, push and pull to all repositories to which the granting user or robot account has write access.
+
+* *Super User Access*.  When selected, provides the user the ability to administer your installation including managing users, managing organizations and other features found in the superuser panel. 
+
+* *Administer User*  When selected, provides the user the ability to  administer your account including creating robots and granting them permissions to your repositories. 
+
+* *Read User Information*.  When selected, provides the user the ability to read user information such as username and email address.
+
+Token distributors should be mindful of the permissions that they are granting when generating a token on behalf of a user, and should have absolute trust in a user before granting such permissions as *Administer organization*, *Super User Access*, and *Administer User*.
+
+OAuth2 access tokens are passed as a `Bearer` token in the `Authorization` header of an API call; this means that they can be used to directly access a {productname} resource, such as an image tag, a repository, an organization, and so on.
+
+With {productname}, there is currently no way to set an expiration time on an OAuth2 access token, and the token lifespan is 10 years. Tokens can be deleted by deleting the applications in which they were created in the event that they are compromised, however, this deletes all tokens that were made within that specific application.
+
+[discrete]
+[id="robot-account-tokens"]
+=== Robot account tokens 
+
+Robot account tokens provide persistent API tokens designed for automation and continuous integration workflows. By default, {productname}'s robot account tokens do not expire and do not require user interaction, which makes robot accounts ideal for non-interactive use cases.
+
+Robot account tokens are automatically generated at the time of a robot's creation and are non-user specific; that is, they are not linked to individual users but are instead associated with a repository or an organization. Robot account tokens provide access without needing a user's personal credentials. How the robot account is configured, for example, with one of `READ`, `WRITE`, or `ADMIN` permissions, ultimately defines the actions that the robot account can take.
```

Comment:
Not entirely correct, they are connected to the user or org namespace where they are created. Every robot account has a prefix which identifies the namespace where it was created. So `ibazulic+robot` means that the robot `robot` is created in the `ibazulic` namespace. Robot accounts get GCed when a namespace (org or private account, doesn't matter) is deleted.